marsha/settings.py

```diff
@@ -151,6 +151,7 @@ class Base(Configuration):
     AWS_ACCESS_KEY_ID = values.SecretValue()
     AWS_SECRET_ACCESS_KEY = values.SecretValue()
     AWS_DEFAULT_REGION = values.Value("eu-west-1")
+    UPLOAD_CONFIRM_SHARED_SECRETS = values.SecretValue()
```

Comment:
You should use a ListValue: https://django-configurations.readthedocs.io/en/latest/values/#configurations.values.ListValue